Source/JavaScriptCore/inspector/scripts/tests/commands-with-async-attribute.json

```diff
@@ -27,6 +27,14 @@
         }
     ],
     "commands": [
+        {
+            "name": "executeSQLAsyncNoReturnValues",
```

Comment:
can we also include the resulting changes to `Source/JavaScriptCore/inspector/scripts/tests/expected/commands-with-async-attribute.json-result`?

Reply:
The protocol generator needs to be rebaselined prior to this change. I will do that so that the generated code changes associated with this PR are obvious.